pkgs/tools/misc/coreutils/default.nix

```diff
@@ -53,7 +53,7 @@ stdenv.mkDerivation rec {
       ++ optional aclSupport acl.crossDrv
       ++ optional attrSupport attr.crossDrv
       ++ optionals selinuxSupport [ libselinux.crossDrv libsepol.crossDrv ]
-      ++ optional (stdenv.ccCross.libc ? libiconv)
+      ++ optional (stdenv.ccCross or {} ? libc.libiconv)
```

Comment:
The `or {}` part is not needed.  If you think `ccCross` might not exist, you can do:

    stdenv ? ccCross.libc.libiconv

I think that is just part of Nix syntax.